cypress/page-objects/student-registration-form.js

```diff
@@ -0,0 +1,153 @@
+class RegistrationFormPage {
+  url = 'https://demoqa.com/automation-practice-form';
+
+  submitButtonSelector = '#submit';
+
+  firstNameInputSelector = '#firstName';
+
+  lastNameInputSelector = '#lastName';
+
+  emailInputSelector = '#userEmail';
+
+  mobileInputSelector = '#userNumber';
+
+  userFormSelector = '#userForm';
+
+  genderSelector = 'input[name="gender"]';
+
+  dateOfBirthDatepickerSelector = '.react-datepicker__input-container';
+
+  dateOfMonthDatepickerSelector = '.react-datepicker__month-select';
+
+  dateOfYearDatepickerSelector = '.react-datepicker__year-select';
+
+  dateOfDayDatepickerSelector =
+    '.react-datepicker__day:not(.react-datepicker__day--outside-month)';
+
+  hobbyCheckboxSelector = 'input[type="checkbox"]';
+
+  hobbyLabelsSelector = '.custom-control-label';
+
+  currentAddressSelector = '#currentAddress';
+
+  stateDropdownSelector = '#state';
+
+  dropdownMenuSelector = 'div[class*="menu"]';
+
+  cityDropdownSelector = '#city';
+
+  subjectsDropdownSelector = '#subjectsContainer';
+
+  dropdownSubjectMenuSelector = 'div[id*="option"]';
+
+  subjectLabelsSelector = '.subjects-auto-complete__multi-value__label';
+
+  selectFileInputSelector = '#uploadPicture';
+
+  get submitBtn() {
+    return cy.get(this.submitButtonSelector);
```

Comment:
To my mind better write ### 
```suggestion
    return cy.get('#submit');
```
and delete part of code at the top

Reply:
did it